src/Builders/OrderBuilder.php

```
@@ -123,12 +123,18 @@ public function create(Menu $menu = null, Customer $customer = null, Currency $c
      */
     public function get(): Collection
     {
-        throw_unless(
-            $subscription = $this->getOption('subscription'),
-            MissingSubscriptionException::class
-        );
+        $customer = $this->getOption('customer');
 
-        $endpoint = "subscriptions/{$subscription->id}/orders";
+        if ($this->getOption('transactional') && $customer) {
+            $endpoint = "customers/{$customer->id}/orders";
+        } else {
```

Tu peux throw un `MissingCustomerException` si on n'a pas de `$customer` et qu'on call le `transactional()`.

Je ne veux pas un silent fail. 😄